### DISCUSSION TITLE
This project is dead ?

### DISCUSSION THREAD

**Discussion by xnohat** [guidance-ai/guidance#377] [Q&A] [answered]
Hi,

This Guidance project is one in just two options existed today for prompting with control flow (another is LMQL). One word for this project: **critical useful**
But why there no maintainers to look at fixing bug or at least merge hundred of PRs already fix many bugs of this project ?
If maintainers too busy to fixing something, so please at least take a bit of time to merge PRs to help this project still running with others.

Many thanks for starting great project,

**Comment by Harsha-Nori** [comment, marked as answer]
Hi @xnohat -- sorry about that! We're working on a significant update to guidance (in this repository's branches) that should address many of the existing issues (e.g. control flow, error handling, etc.) and also give us a strong foundation to develop the next generation of the library. We'll be saying more about this in the near future.

Just want to stress that guidance isn't dead. We are actively at work, and we really appreciate your (and everyone's!) patience as we get guidance ready for its next stage!

**Reply by xnohat** [reply]
Thanks and appreciate your updating
